tools/ci/packaging/archlinux/build.sh

```diff
@@ -15,6 +15,7 @@ sed -i "s|/A3ULPATH|$A3UL_DIR|g" PKGBUILD
 
 mkdir src
 cp -RL /$A3UL_DIR/* ./src
+chown -R builduser:builduser .
```

Comment:
:thinking: Maybe you can try to use:
`chown -R builduser:builduser "$SELF_PATH"`

It will do the same but doesn't use the dot :wink:

Reply:
Oops you seem to change directory again, closest seems to refer to `$BUILD_DIR` so use that instead...

Reply:
Although IMHO seeing what you actually want to accomplish, i would suggest to change the group ownership and allow `rwX` for that group recursively on `$BUILD_DIR`.
Thus more like:
```
chgrp -R builduser $BUILD_DIR
chmod -R g+rwX $BUILD_DIR
```